cardea-data/src/main/java/ca/on/oicr/gsi/cardea/data/Case.java

```diff
@@ -55,7 +55,7 @@ private Case(Builder builder) {
         .filter(sample -> sample.getRequisitionId() != null
             && sample.getRequisitionId().longValue() == builder.requisition.getId())
         .map(Sample::getCreatedDate)
-        .min(LocalDate::compareTo).orElse(null);
+        .min(LocalDate::compareTo).orElse(builder.startDate);
```

Comment:
If the builder has a start date, we can trust that it was calculated correctly - check if builder.startDate is non-null and use that value first if so. Only calculate if it's null